test/topology_experimental_raft/test_tablets.py

```diff
@@ -1222,3 +1240,206 @@ async def test_tablet_storage_freeing(manager: ManagerClient):
     logger.info("Verify that the table's disk usage on first node shrunk by about half.")
     size_after = await manager.server_get_sstables_disk_usage(servers[0].server_id, "test", "test")
     assert size_before * 0.33 < size_after < size_before * 0.66
+
+@pytest.mark.asyncio
+async def test_decommission_rack_basic(manager: ManagerClient):
+    """
+    Test decommissioning of all nodes in a rack
+    when there are enough remaining racks to satisfy
+    the replication factor.
+    """
+    logger.info("Bootstrapping multi-rack cluster")
+    servers = dict[ServerNum, ServerInfo]()
+    num_racks = 3
+    nodes_per_rack = 2
+    shards_count = 2
+    initial_tablets = 64
+    rf = num_racks - 1
+    ks = "test"
+    table = "test"
+    for rack in range(1, num_racks + 1):
+        rack_servers = await manager.servers_add(nodes_per_rack, property_file={"dc": "dc1", "rack": f"rack{rack}"})
+        for s in rack_servers:
+            servers[s.server_id] = s
+
+    logger.info("Populate data")
+    cql = manager.get_cql()
+    await cql.run_async(f"CREATE KEYSPACE test WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': {rf}}} AND tablets = {{'initial': {initial_tablets}}};")
```

Comment:
Another possible utility function: `ks, table = create_table(rf, initial_tablets)`, that way we even get rid of the static at the beggining of each test
```
    ks = "test"
    table = "test"
```

Sidenote: if initail_tablets are not important for this test, we could set it as a defaut in the util function and that would be one more static line hat would not need to present for each test

Reply:
The test needs to know it for the assertions below, so I don't want it to double guess the default value, but rather pass the arbitrary value to the create function so we have one authoritative value.

Reply:
Test does need to know it, but it doesnt care about its value, it is not important for the testcase, so an utility function or fixture can be used and it can return the `ks` and `table` with some value, which we can use in the test

Reply:
done in next version